src/main/java/webserver/request/HTTP_HEADERS.java

```diff
@@ -10,4 +10,12 @@
   public static final String GET = "GET";
   public static final String HEAD = "HEAD";
   public static final String POST = "POST";
+
+  //status codes
+  public static final int STATUS_200 = 200;
```

Comment:
I don't have any objections to these names, but it might clarify intent if you say something like `STATUS_OK`, `STATUS_CREATED`, etc...